otter/convergence/steps.py

```diff
@@ -300,8 +297,7 @@ def as_effect(self):
         # Since we're deleting a node, we'll ignore any errors which indicate
         # that the node doesn't exist.
         return eff.on(
-            error=_ignore_errors(
-                CLBDeletedError, NoSuchCLBError, NoSuchCLBNodeError)
+            error=_ignore_errors(CLBNotFoundError)
```

Comment:
Ah! I think you found the reason the integration tests are failing! Yeah, this was a simple typing mistake, I accidentally deleted too much...